Makefile

```diff
@@ -73,7 +73,7 @@ lint: lint-go lint-ts lint-prettier
 
 .PHONY: lint-go
 lint-go:
-	golangci-lint run
+	golangci-lint run --timeout 2m 
```

Comment:
This has been a problem on CI recently for some reason. I'm not 100% sure why (it started to occur after we switched to `go 1.14`). Let me know if you disagree with this change @albrow.

Reply:
👍 this is okay with me.